doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -207,6 +207,7 @@ nRF5340 Audio
 * Updated:
 
   * The application to use the ``NFC.TAGHEADER0`` value from FICR as the broadcast ID instead of using a random ID.
+  * Change from Newlib to Picolib to align with NCS and Zephyr
```

Comment:
```suggestion
  * The application to change from Newlib to Picolib to align with |NCS| and Zephyr.
```

Reply:
@umapraseeda , this looks a bit strange to me. The wording sounds like it will change, but this PR in fact changes it.

Reply:
What about "The application has changed ..."

Reply:
Usually it is read along with the 'Updated' bullet header: So, read as, 'Updated the application to change from Newlib to Picolib to align with |NCS| and Zephyr.' Another option: The application now uses Picolib instead of Newlib to ...